src/controller.py

```diff
@@ -20,6 +21,26 @@ def log(text: str, color: str = LOG_COLOR) -> None:
     helpers.log(f"CONTROLLER: {text}", color)
 
 
+REGTEST_RPC = BTCJsonRPC(
+    url="http://0.0.0.0:18021",
+    user="rpc",
+    passwd="rpc",
+)
+# Needed for setup purposes
+# There is no wallet by default, accounting for possible restart
+#   of the controller, in which case it will already have the wallet
+# TODO: think about some better solution how to make setup
+try:
+    REGTEST_RPC.createwallet("tenv_wallet")
+    log("Wallet created")
+except Exception as err:
+    if "Database already exists" not in str(err):
+        raise
+    log("Wallet already exists")
+FEES_PER_KVB = 0.00001  # 1 sat/vB
+REGTEST_RPC.settxfee(FEES_PER_KVB)
```

Comment:
I think we should do this outside of tenv. @vdovhanych could we put this into the Dockerfile directly? Simply RUN some commands at the end?

Reply:
Agreed, it would be nice, also from the point that this setup process in `controller` fails when the `regtest` container is not running. We would need some error handling, which could be avoided by doing that in the `regtest` itself